Google.Api.Gax.Grpc/ChannelPool.cs

```diff
@@ -44,8 +46,11 @@ public sealed class ChannelPool
         /// if they require any.
         /// </summary>
         /// <param name="scopes">The scopes to apply. Must not be null, and must not contain null references. May be empty.</param>
-        public ChannelPool(IEnumerable<string> scopes)
+        /// <param name="useJwtWithScopes"></param>
```

Comment:
Please fill in the documentation comment here.

Reply:
done, thanks!